ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkFields/NetworkFields.test.tsx

```diff
@@ -27,31 +39,16 @@ import {
 
 const mockStore = configureStore();
 
-const changeField = async (
-  wrapper: ReactWrapper,
-  selector: string,
-  name: string,
-  value: unknown
-) => {
-  return act(async () => {
-    wrapper.find(selector).simulate("change", {
-      target: {
-        name,
-        value,
-      },
-    });
-  });
-};
-
 describe("NetworkFields", () => {
   let state: RootState;
 
   beforeEach(() => {
+    const vlan = vlanFactory({ fabric: 1, vid: 1 });
     state = rootStateFactory({
       fabric: fabricStateFactory({
         items: [
-          fabricFactory({ id: 1, default_vlan_id: 1 }),
-          fabricFactory({ default_vlan_id: 1 }),
+          fabricFactory({ id: 1, default_vlan_id: vlan.id }),
```

Comment:
For some reason when I migrated this file to RTL it introduced a leak between the tests. I tracked it down to setting up the store in two concurrent tests but I couldn't figure out why the leak was occurring (the leak was happening even if the only thing in the "it(...)" block was creating the state).

I'd love to hear any suggestions about why that leak might be occurring. In the meantime, using unique ids model ids appears to make the tests pass.

Reply:
What do you mean by "a leak between the tests"? If you mean that state in one test affects state in another, it could be the use of `sequence` for model ids. It increments by one every time it's invoked so maybe there can be overlaps when working with low id numbers? Maybe we should just be using `random`?